cypress/integration/profile_page_spec.js

```diff
@@ -72,10 +74,9 @@ describe("Profile Page", () => {
     expect($post).to.contain("Show this message")
     expect($post).not.to.include.text("Do not display")
   })
-  // cy.get(".posts").should("contain", "Show this message")
-  // cy.get(".posts").should("not.contain", "Do not display")
   
-  // drop Posts collection from database
+  // use webhelper to drop users and posts collections
   cy.task("dropPosts");
+  cy.task("dropUsers");
```

Comment:
you can use after instead of afterEach if you don't want the db to be dropped between tests